babel-plugin-strip-wrappers.js

```diff
@@ -0,0 +1,57 @@
+const syntaxDecorators = require('@babel/plugin-syntax-decorators').default;
```

Comment:
Where does this actually come from? I think we need to be a bit more specific here. This currently expects the "top level" version to be what we want, but I think in reality we want the one from ember-cli-babel itself, right?

Reply:
Oh yeah, I guess it's pretty accidental:

```sh
~/projects/ember-test-waiters$ yarn why @babel/plugin-syntax-decorators
yarn why v1.21.1
[1/4] 🤔  Why do we have the module "@babel/plugin-syntax-decorators"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@babel/plugin-syntax-decorators@7.7.4"
info Reasons this module exists
   - "documentation#@babel#plugin-proposal-decorators" depends on it
   - Hoisted from "documentation#@babel#plugin-proposal-decorators#@babel#plugin-syntax-decorators"
info Disk size without dependencies: "20KB"
info Disk size with unique dependencies: "48KB"
info Disk size with transitive dependencies: "48KB"
info Number of shared dependencies: 1
✨  Done in 0.55s.
```

And `documentation` is a `devDependency`, so at runtime we'll be getting it from 🤷‍♂ 

Are you suggesting something like

```javascript
const emberCliBabelPath = require.resolve('ember-cli-babel');
const syntaxDecoratorsPluginPath = require.resolve('@babel/plugin-syntax-decorators', {
  paths: [ emberCliBabelPath ]
});
```

?